modules/vsphere/vsphere_test.go

```diff
@@ -300,7 +300,7 @@ func TestVSphere_Collect(t *testing.T) {
 
 	//working around ioutil.TempDir naming convention
 	for key, value := range collected {
-		match,_ := regexp.MatchString("*LocalDS_*",key)
+		match,_ := regexp.MatchString(".*LocalDS_.*",key)
```

Comment:
no need in using regexp, [`strings.Contains`](https://golang.org/pkg/strings/#Contains) will doo